activerecord/lib/active_record/railties/query_comments.rb

```diff
@@ -0,0 +1,70 @@
+module ActiveRecord
+  module Railties # :nodoc:
+    module QueryComments #:nodoc:
+      module ActionController
+        extend ActiveSupport::Concern
+
+        included do
+          if Rails.configuration.action_controller.query_comments_action_filter_enabled
+            class_eval do
```

Comment:
Does this work without the class eval? I'd think we'd already be at the class level inside this `included` block.

Reply:
Hmm I can't remember what led me to using this approach - I'll retest it all without, as I agree it feels like it's unnecessary :)

Reply:
Still can't remember what problem this solved - might have just been a red herring. Integration tests in an app pass fine without this unnecessary nesting 👍